src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/DuplicateReferenceValidation.feature

```diff
@@ -0,0 +1,270 @@
+                  }
+                ]
+               }
+              """
+              Then it should respond with 201 or 200
+
+        Scenario: 03 Verify clients can not create a assessment resource with combined duplicate descriptors
+            
+              When a POST request is made to "/ed-fi/assessments" with
+              """
+              {
+                "assessmentIdentifier": "01774fa3-06f1-47fe-8801-c8b1e65057f2",
+                "namespace": "uri://ed-fi.org/Assessment/Assessment.xml",  
+                "assessmentTitle": "3rd Grade Reading 1st Six Weeks 2021-2022",
+                "academicSubjects": [
+                  {
+                    "academicSubjectDescriptor": "uri://ed-fi.org/AcademicSubjectDescriptor#English Language Arts"
+                  }
+                ],
+                "performanceLevels": [
+                  {
+                    "performanceLevelDescriptor": "uri://ed-fi.org/PerformanceLevelDescriptor#Advanced",
+                    "assessmentReportingMethodDescriptor": "uri://ed-fi.org/AssessmentReportingMethodDescriptor#Scale score",
+                    "minimumScore": "23",
+                    "maximumScore": "26"
+                  },
+                  {
+                    "performanceLevelDescriptor": "uri://ed-fi.org/PerformanceLevelDescriptor#Advanced",
+                    "assessmentReportingMethodDescriptor": "uri://ed-fi.org/AssessmentReportingMethodDescriptor#Scale score",
+                    "minimumScore": "27",
+                    "maximumScore": "30"
+                  } 
+                ],
+                "scores": [
+                  {
+                    "assessmentReportingMethodDescriptor": "uri://ed-fi.org/AssessmentReportingMethodDescriptor#Raw score",
+                    "maximumScore": "10",
+                    "minimumScore": "0",
+                    "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer"
+                  }
+                ]
+               }
+              """
+              Then it should respond with 400
+              And the response body is
+              """
+              {
+                "detail": "Data validation failed. See 'validationErrors' for details.",
+                "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                "title": "Data Validation Failed",
+                "status": 400,
+                "correlationId": null,
+                "validationErrors": {
+                    "$.performanceLevels[*].assessmentReportingMethodDescriptor": [
+                        "The 2nd item of the performanceLevels has the same identifying values as another item earlier in the list."
+                    ],
+                    "$.performanceLevels[*].performanceLevelDescriptor": [
+                        "The 2nd item of the performanceLevels has the same identifying values as another item earlier in the list."
+                    ]
```

Comment:
Would it be difficult to get the following instead, matching the ODS/API?

```json
"validationErrors": {
    "$.performanceLevels": [
      "The 2nd item of the AssessmentPerformanceLevels has the same identifying values as another item earlier in the list."
    ]
  }
```

Reply:
@stephenfuqua 
As discussed with @bradbanister, the DMS error message is more accurate because it clearly explains what is wrong with the request document.